Makefile

```diff
@@ -93,15 +120,9 @@ generate-clientset:
 	GO111MODULE=on $${GOPATH}/src/k8s.io/code-generator/generate-groups.sh \
 		client,deepcopy \
 		github.com/submariner-io/submariner-operator/pkg/client \
-		github.com/submariner-io/submariner-operator/pkg/apis \
+		github.com/submariner-io/submariner-operator/apis \
```

Comment:
Why do we move the apis out of pkg?  is it necessary?

Reply:
That’s what the SDK expects now...

Reply:
I followed the migration guide and this is how the directories layout should look like. I think we can choose the directories by changing the controller-gen parameters and kustomize configuration